types/src/block/mod.rs

```diff
@@ -402,21 +397,25 @@ impl BlockHeader {
     }
 
     //for test
-    pub fn dag_genesis_random() -> Self {
```

Comment:
why not use [cfg(test)]

Reply:
[cfg(test)] cannot be used by other modules.

Reply:
got it. But I think function dag_genesis_random and dag_genesis_random_with_parent only use in flexidag/tests/test.rs, can we move it to releated files. Maybe it does't need to make public to other mods.

Reply:
Ok, I will try.